lrpar/examples/mut_param/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "mut_param"
+version = "0.1.0"
+authors = ["Laurence Tratt <http://tratt.net/laurie/>"]
```

Comment:
I think your name should go here!

Reply:
Should be resolved in [bf4ada3](https://github.com/softdevteam/grmtools/pull/471/commits/bf4ada313fdf1a78b517d5d872f5a563718ff57e).